### DISCUSSION TITLE
name 'params' is not defined

### DISCUSSION THREAD

**Discussion by jackbrougher** [theislab/scvelo#441] [Q&A] [answered]
Hello,

I'm running through scVelo for the first time following the tutorial but with my own data set. I'm getting to the Basic Preprocessing step and then getting a NameError that params isn't defined. I'm running scv.set_figure_params() before this, am I missing something else?

![image](https://user-images.githubusercontent.com/83614212/117498588-d9101b80-af2e-11eb-81bb-c0c2de4cf20f.png)

**Comment by VolkerBergen** [comment, marked as answer]
`params` here is just a placeholder for keyword arguments that you could pass, e.g., `min_shared_counts=30`. The "Getting Started" section is really just to get an idea of how things work. Go for https://scvelo.readthedocs.io/VelocityBasics.html, which is an actual tutorial that you can follow to do the analysis on your own data.
